### PR TITLE
Add attestation support to playback API

### PR DESCRIPTION
**Pull request by leszko**
**What does this pull request do? Explain your changes. (required)**

Add support for returning playback info form `/api/playback/<attestation-id>`.

**Specific updates (required)**

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

VID-211

**Screenshots (optional)**

**Checklist**

- [x] I have read the **CONTRIBUTING** document.
- [ ] ~My change requires a change to the documentation.~
- [ ] ~I have updated the documentation accordingly.~
- [x] I have added tests to cover my changes.

### CODE REVIEW

**Review by iameli — APPROVED** [review]
LGTM

File: packages/api/src/controllers/playback.ts [hunk 1]
```diff
@@ -114,7 +115,7 @@ export async function getResourceByPlaybackId(
     if (asset.status.phase !== "ready") {
       throw new UnprocessableEntityError("asset is not ready for playback");
     }
-    if (asset.userId !== user?.id && !isCrossUserQuery) {
+    if (asset.userId !== user?.id && !cutoffDate) {
```
Comment (victorges) [inline packages/api/src/controllers/playback.ts]: I think this condition is inverted now

File: packages/api/src/schema/api-schema.yaml [hunk 2]
```diff
@@ -1929,6 +1929,9 @@ components:
                   bitrate:
                     type: number
                     example: 449890
+            attestation:
+              $ref: "#/components/schemas/attestation"
```
Comment (victorges) [inline packages/api/src/schema/api-schema.yaml:1933]: Hmm is this like a circular reference?

**Review by victorges — APPROVED** [review]
LGTM

File: packages/api/src/controllers/playback.ts (hunk 1 above)
Comment (leszko) [reply packages/api/src/controllers/playback.ts]: Right 🤦 Updated in 15337119dbab5508deb71934225b18c4c5a38b08

File: packages/api/src/schema/api-schema.yaml (hunk 2 above)
Comment (leszko) [reply packages/api/src/schema/api-schema.yaml:1933]: No, it points correctly here: https://github.com/livepeer/studio/blob/42aa0e87e9869f7b29396efefa10ab22b6e6ab91/packages/api/src/schema/db-schema.yaml#L371

Comment (victorges) [reply packages/api/src/schema/api-schema.yaml:1933]: Ohh I only realized now that this was a field and not a component schema 😅

**Pull request merged**